### ISSUE TITLE
SEGFAULT while drag-resizing a window via corner

### ISSUE THREAD

**Issue by voidburn** [glfw/glfw#905]
When the `glfwSetFramebufferSizeCallback` is assigned a function that also executes a `glfwSwapBuffers` and both width and height are affected at the same time, the program crashes with a segfault.

Reproducible with the `splitview` example in the latest 3.2.1 release.

Video of the circular resize motion that triggers the crash (In the video Lwjgl3 is being used to create the window, the same motion causes the splitview example to crash) (https://github.com/LWJGL/lwjgl3/files/608741/2016-11-23.12-47-48.zip)

See: https://github.com/LWJGL/lwjgl3/issues/254 for more detailed description and reproduction steps

**Comment by elmindreda** [comment]
Which platform are you on?

**Comment by voidburn** [comment]
Windows 10 64bit, but after testing on several computers and not being able to reproduce it, I started looking inwards... into other processes in my machine...

Turns out that what was interfering with GLFW was Asus's Sonic Suite, which was reinstalled automatically with the latest driver updates. Somehow that thing hooks itself into everything (probably for that bullshit game radar feature thingy). Removed that piece of garbage and everything went back to a working state.
